traveler-ios-ui/TravelerKitUI/UIUtilities/SearchStateButton.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  SearchStateButton.swift
+//  TravelerKitUI
+//
+//  Created by Ben Ruan on 2019-11-27.
+//  Copyright © 2019 GuestLogix. All rights reserved.
+//
+
+import UIKit
+
+open class SearchStateButton: UIButton {
```

Comment:
You can make this into a more generic state button (BinaryStateButton for example) and add two properties disabledTitle and enabledTitle with default values.  This way it can be reused in the future.

Reply:
Same thing for the images

Reply:
It turns out we don't need a custom class for the button so I removed this class.